src/main/java/org/venice/beachfront/bfapi/services/converter/ShapefileConverter.java

```diff
@@ -110,8 +114,8 @@
 			// Return the results
 			result = java.nio.file.Files.readAllBytes(zipFile.toPath());
 
-			cleanup(shapefile);
-			cleanup(zipFile);
+			// Cleanup the temporary directory
+			FileSystemUtils.deleteRecursively(tempDirectory);
```

Comment:
According to the docs, DataStore can have live connections to files. http://docs.geotools.org/latest/userguide/library/data/datastore.html
This may not delete everything while dataStore is in scope.

Reply:
Added Dispose call